gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -539,6 +543,7 @@ def parse(self, *args, **kwargs) -> bool:
         if hasattr(self, "dataset_hash") and self.dataset_hash and not self.dry_run:
             self.info("Published dataset's IPFS hash is " + str(self.dataset_hash))
 
+        # TODO: Is anything relying on this return value? If return value is invariant, may as well not return a value.
```

Comment:
We can nix the boolean but I’d like to return a message of some sort indicating success. That’s of minor utility during development and testing.

Reply:
Return or log?

Reply:
I’m fine with a log statement